containers/datalab/content/GCSbackup.sh

```diff
@@ -139,9 +139,11 @@ echo
 echo "${timestamp}: Running GCS backup tool.." | tee -a ${log_file}
 
 # create an archive of the backup path
-archive_name=$(mktemp -d)"/archive.zip"
+tempdir="$(mktemp -d)"
+archive_name="${tempdir}/archive.zip"
+trap "rm -rf ${tempdir}" EXIT
 echo "Creating archive: $archive_name"
-zip -rq ${archive_name} "${backup_path}" || {
+zip -rq ${archive_name} "${backup_path}" -x '*/.forever/*' || {
```

Comment:
I think you have an extra "/" here at the beginning that you don't want. Did you mean "\"?

Reply:
I did intend for the slash to be there because I wanted to exclude the "/content/.forever" dir, but not anything like "/content/this.directory.should.be.backed.up.forever".

Hence, the leading slash.

I tested this locally with a ".forever" directory and confirmed that adding/removing/changing files under that directory did not alter the contents of the backup.

Is there an issue you think the leading slash will cause?

Reply:
Sorry, I was testing this when actually in the directory, like:

$ cd /content
$ zip -rq archive.zip . -x '*/.forever/*'

In which case it would show up as ".forever" and wouldn't match, but here we don't cd to /content first, so it would be "/content/.forever". So your way is all good. Carry on!